tests/testthat/test-validate.R

```diff
@@ -175,6 +175,7 @@ test_that("No error is thrown if an input value equals a bound", {
     risk_type = "trisk",
     settlement_factor = 1,
     scc = 40,
+    market_passthrough = 0,
```

Comment:
it would be good to add this variable to all tests for concistency

Reply:
done